.github/ISSUE_TEMPLATE/team_issues.yaml

```diff
@@ -0,0 +1,43 @@
+name: Team issue template
+description: This template is reserved for team members.
+title: "Keep the title concise. Use the common language we can use to identify the issue."
+body:
+  - type: textarea
+    attributes:
+      label: What?
+      description: A concise (or detailed) description of this issue's goal.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Why?
+      description: A concise (or detailed) explanation of why we need this issue.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Suggestion
+      description: Do you have a suggestion to close the issue?
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Tasks
+      description: Feel free to remove the tasklist items that are not relevant for this issue.
+      value: |
+        ```[tasklist]
+        ### Tasks
+        - [ ] Subissue
```

Comment:
Should this suggest a possible list of subissues? E.g.:
```
- [ ] Subissue 1
- [ ] Subissue 2
- [ ] Subissue i
```
Or something similar? This does not seem the best option either :thinking:

Reply:
I got your intention, and it’s a good idea. Since this is an internal issue only for the team, IDK if it matters much. A single item can be enough for us to remember to put issues there. WDYT?

Reply:
Right, if we can keep this internal it is OK for me :+1: